doc/3/controllers/document/update/snippets/update.java

```diff
@@ -0,0 +1,7 @@
+    ConcurrentHashMap<String, Object> content = new ConcurrentHashMap<>();
+    content.put("name", "Johny");
+
+    ConcurrentHashMap<String, Object> response =
+    kuzzle.getDocumentController().update("nyc-open-data", "yellow-taxi", "some-id", content)
+    .get();
+    System.out.println(response);
```

Comment:
Can you add the result of the `println` in comment? 
It helps developer to better apprehend the structure of the response object they have. We try to do this for every method (Look the [Javascript SDK](https://github.com/kuzzleio/sdk-javascript/blob/master/doc/7/controllers/document/create/snippets/create.js) and [Doc](https://docs.kuzzle.io/sdk/js/7/controllers/document/create/#usage))